problem-16.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdbool.h>
+// Basic calculator program that performs addition, subtraction, multiplication and division
+
+int getSum(int firstNum, int secondNum);
+int getDifference(int firstNum, int secondNum);
+int getProduct(int firstNum, int secondNum);
+int getQuotient(int firstNum, int secondNum);
+
+int main()
+{
+    int firstNum, secondNum, user_choice;
+    printf("---------CALCULATOR---------\n");
+
+    printf("First number = ");
+    scanf("%d", &firstNum);
+
+    printf("Second Number = ");
+    scanf("%d", &secondNum);
+
+    printf("\n1. Addition");
+    printf("\n2. Subtraction");
+    printf("\n3. Multiplication");
+    printf("\n4. Division");
+
+    while (true)
+    {
+        printf("\n\nEnter your choice: ");
+        scanf("%d", &user_choice);
+
+        if (user_choice == 1)

```

Comment:
How about we use [enum](https://www.w3schools.com/c/c_enums.php) here instead of directly compare to `1`? Currently its ambiguous but enum will add clarity in it.